src/rust/ensogl/lib/core/src/control/io/mouse/event.rs

```diff
@@ -47,6 +49,37 @@ macro_rules! define_events {
             pub fn button(&self) -> mouse::Button {
                 mouse::Button::from_code(self.raw.button().into())
             }
+
+            /// Return the position relative to the event handler that was used to catch the event.
+            /// If the event handler does not have a position in the DOM, the returned position
```

Comment:
Please explain it better - when this could happen "If the event handler does not have a position in the DOM" - describe the example with Window .